ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -1837,5 +1837,41 @@
     "FieldType": "OthersSelector",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2018",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "reviewCMOOthersHeader",
+    "FieldType": "Label",
+    "Label": "### Select recipients",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "reviewCMOOthersSubHeader",
+    "FieldType": "Label",
+    "Label": "You must check to see if there are restrictions on who should be notified about the orders.",
+    "SecurityClassification": "PUBLIC",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "reviewCMONotifyAllOthers",
+    "Label": "Send the orders to all others in the case?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
```

Comment:
these fields are very similar to the one's already defined.. otherDetailsSectionSubHeader, otherDetailsSectionHeader, sendOrderToAllOthers... would be good if they could be reused but if they need to differ slightly then fine to leave. :)

Reply:
updated to reuse the existing fields